src/components/organisms/Nav.js

```diff
@@ -132,14 +183,42 @@ const Nav = () => {
             confirmText="Sign Up"
             cancelHandler={() => setModalState(modalStates.closed)}
           >
+            <label htmlFor="name">
+              Display Name
+              <input
+                type="text"
+                name="name"
+                value={usernameInput}
+                onChange={() => { handleNameChange(); }}
+                onBlur={() => { validation(); }}
+                required
+              />
+            </label>
+              {nameError}
```

Comment:
the Modal component expects all its children to be html elements. Since this is a string, it's giving off a props error when loaded. Just toss these in <p> tags and that should fix the issue.
```suggestion
              <p>{nameError}</p>
```